lib/elixir/lib/calendar.ex

```diff
@@ -0,0 +1,87 @@
+defmodule Calendar do
```

Comment:
The idea of having `use Calendar` would only be temporary to support folks migrating from Calendar. Then later on we can make it whatever we want.